1-js/02-first-steps/09-alert-prompt-confirm/article.md

```diff
@@ -1,109 +1,108 @@
-# Interaction: alert, prompt, confirm
+# Interação: alert, prompt, confirm
 
-This part of the tutorial aims to cover JavaScript "as is", without environment-specific tweaks.
-
-But we'll still be using the browser as our demo environment, so we should know at least a few of its user-interface functions. In this chapter, we'll get familiar with the browser functions `alert`, `prompt` and `confirm`.
+Esta parte do tutorial tem como objetivo cobrir o JavaScript  "como é ", sem ajustes específicos de ambiente.
+Mas ainda estaremos usando o navegador como nosso ambiente de demostração, portanto devemos conhecer pelo menos algumas de suas funções de interface. Neste capítulo, familiazaremos com as funções do navegador `alert`, `prompt` e `confirm`.
 
 ## alert
 
-Syntax:
+Sintaxe:
 
 ```js
-alert(message);
+alert(mensagem);
 ```
 
-This shows a message and pauses script execution until the user presses "OK".
+ mostrará uma mensagem até o usuário pressionar  "OK".
```

Comment:
This phrase should start with capital letter, and I think you should set the verb on present: "Mostra uma mensagem".